uber/site_sections/registration.py

```diff
@@ -553,11 +560,6 @@ def new(self, session, show_all='', message='', checked_in=''):
         if 'reg_station' not in cherrypy.session:
             raise HTTPRedirect('new_reg_station')
 
-        groups = set()
-        for a in session.query(Attendee).filter(Attendee.first_name == '', Attendee.group_id != None, Attendee.badge_status == c.NEW_STATUS) \
-                                        .options(joinedload(Attendee.group)).all():
-            groups.add((a.group.id, a.group.name or 'BLANK'))
-
```

Comment:
This was no longer being used in the template; we now link them to the main prereg checkin page for claiming group badges.